website/api/zkvm/precompiles.md

```diff
@@ -33,24 +34,44 @@ git = "https://github.com/risc0/RustCrypto-hashes"
 tag = "sha2-v0.10.6-risczero.0"
 ```
 
+Make sure that your dependency gives the same version of the crate as listed in
+the patch tag. In this example, the dependency would be
+```toml
+[dependencies]
+sha2 = "0.10.6"
+```
+
+In some situations, particularly for crates you've used before with patch
+version updates, you may need to regenerate your `lock` file. For example:
+```toml
+cargo update -p sha2 --precise 0.10.6
+```
+
 When using cryptography indirectly, e.g. via the `cookie`, `oauth2`, or `revm`,
 crates it may be possible to enable acceleration support without code changes by
 applying a [Cargo patch][cargo-patch].
 
+Our elliptic curve, keccak, and RSA precompiles still undergoing revision and
```

Comment:
Let's move forward with this assuming we'll land it before we land keccak. 

Also, I wonder if we can write this in a way that we won't have to manage granular updates about which precompiles are marked unstable... might be easier to just accept the maintenance work as we remove those flags (and add other precompiles marked similarly) but figured i'd mention the idea...

Reply:
I remove the keccak wording in #2642 because it was not yet released, sounds like I should add it back proactively just so we don't forget when we add keccak to the table?